src/core/providers/canvas-view-settings/canvas-view-settings.context.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {
+  CanvasViewSettingsContextModel,
+  ceateInitialSettings
+} from './canvas-view-settings.model';
+
+export const CanvasViewSettingsContext =
+  React.createContext<CanvasViewSettingsContextModel>({
+    canvasViewSettings: ceateInitialSettings(),
+    zoomIn: () => {},
+    zoomOut: () => {}
+  });
```

Comment:
You can set this to null

```diff
export const CanvasViewSettingsContext =
-  React.createContext<CanvasViewSettingsContextModel>({
+  React.createContext<CanvasViewSettingsContextModel | null>(
- {
-    canvasViewSettings: ceateInitialSettings(),
-    zoomIn: () => {},
-   zoomOut: () => {}
-
-  });
null);
```

The on the user Context add a try catch

Reply:
It's fixed